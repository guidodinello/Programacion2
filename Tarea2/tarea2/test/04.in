# Implementar finalCadena, anterior
insertarAlFinal (10,2.1)
finalCadena
infoCadena
insertarAlFinal (9,2.1)
insertarAlFinal (8,2.1)
insertarAlFinal (7,2.1)
insertarAlFinal (6,2.1)
insertarAlFinal (5,2.1)
insertarAlFinal (4,2.1)
finalCadena
infoCadena
anterior
infoCadena
anterior
infoCadena
anterior
infoCadena
anterior
infoCadena
anterior
infoCadena
anterior
infoCadena
Fin
