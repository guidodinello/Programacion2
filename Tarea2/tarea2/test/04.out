1>#  Implementar finalCadena, anterior.
2>Insertado al final.
3>loc al final.
4>(10,2.10)
5>Insertado al final.
6>Insertado al final.
7>Insertado al final.
8>Insertado al final.
9>Insertado al final.
10>Insertado al final.
11>loc al final.
12>(4,2.10)
13>loc al anterior.
14>(5,2.10)
15>loc al anterior.
16>(6,2.10)
17>loc al anterior.
18>(7,2.10)
19>loc al anterior.
20>(8,2.10)
21>loc al anterior.
22>(9,2.10)
23>loc al anterior.
24>(10,2.10)
25>Fin.
